librz/core/canalysis.c

```diff
@@ -446,33 +451,40 @@ static void bb_info_print(RzCore *core, RzAnalysisFunction *fcn, RzAnalysisBlock
 		pj_ki(pj, "ninstr", bb->ninstr);
 		pj_kb(pj, "traced", bb->traced);
 		pj_end(pj);
+		char *json_str = rz_str_dup(pj_string(pj));
+		pj_free(pj);
+		rz_strbuf_append(buf, json_str);
 		break;
 	}
 	case RZ_OUTPUT_MODE_TABLE:
```

Comment:
It broke tables output:
```
[XX] db/cmd/tables tables sum
RZ_NOPLUGINS=1 /Users/runner/bin/rizin -escr.utf8=0 -escr.color=0 -escr.interactive=0 -eflirt.sigdb.load.system=false -eflirt.sigdb.load.home=false -N -Qc 's main
af
afbt
afbt:fancy
afbt:size/sort/inc:simple
' bins/elf/hello_world
-- stdout
--- expected
+++ actual
@@ -1,17 +1,29 @@
-addr  size jump  fail  
------------------------
-0x7aa 92   0x844 0x806
-0x806 62   0x844 -1
-0x844 3    -1    -1
-.------------------------------.
-| addr  | size | jump  | fail  |
-)------------------------------(
-| 0x7aa | 92   | 0x844 | 0x806 |
-| 0x806 | 62   | 0x844 | -1    |
-| 0x844 | 3    | -1    | -1    |
-`------------------------------'
-addr  size jump  fail  
------------------------
-0x844 3    -1    -1
-0x806 62   0x844 -1
-0x7aa 92   0x844 0x806
+addr size jump fail 
+--------------------
+addr size jump fail 
+--------------------
+addr size jump fail 
+--------------------
+
+addr size jump fail 
+--------------------
+addr size jump fail 
+--------------------
+addr size jump fail 
+--------------------
+addr size jump fail 
+--------------------
+
+.---------------------------.
+| addr | size | jump | fail |
+)---------------------------(
+`---------------------------'
+addr size jump fail 
+--------------------
+addr size jump fail 
+--------------------
+addr size jump fail 
+--------------------
+
+addr size jump fail 
+--------------------
```

Reply:
Sure I will take a look on weekend and fix it, I had tried to fix it multiple times but it didn't help. On Weekend, I will try again.